### DISCUSSION TITLE
Is there any good solution to mirror all repositories from github sass to GHES

### DISCUSSION THREAD

**Discussion by zhouwenyang** [community/community#38450] [Enterprise]
Is there any good solution to mirror all repositories from github sass to GHES. 
I can find the solution with CLI. https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository
but I want find a automatic solution.

**Comment by djdefi** [comment]
One option using GitHub Actions is https://github.com/repo-sync/repo-sync -- This is designed for single repositories, but could possibly be adapted or used as inspiration for a larger mirroring setup.
